tests/lib/snaps/store/test-snapd-mount-control-nfs/bin/sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+PS1='$ '
+
+exec "$@"
```

Comment:
This is not `/bin/sh "$@"`, the name is misleading and it differs from other "sh" apps in other tests. If you want to keep the ".cmd" suffix and semantics just do so.